bootstrap.sh

```diff
@@ -17,4 +17,4 @@ kubectl apply -f .infrastructure/app/deployment.yml
 
 # Install Ingress Controller
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
-# kubectl apply -f .infrastructure/ingress/ingress.yml
+kubectl apply -f .infrastructure/ingress/ingress.yml
```

Comment:
Ensure that the path `.infrastructure/ingress/ingress.yml` is correct and that the file exists in the specified location. The path should match the actual file structure in your project.